### PR TITLE
[three] r129 minor fixes

### PR DESCRIPTION
**Pull request by joshuaellis**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/mrdoob/three.js/blob/r129/examples/jsm/postprocessing/EffectComposer.js#L318 & https://github.com/mrdoob/three.js/blob/a2b92e5260a2a973376e25fd5212b7604574816b/src/Three.js#L110
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/three/index.d.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // Definitions by: Josh Ellis <https://github.com/joshuaellis>
 //                 Nathan Bierema <https://github.com/Methuselah96>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.6
+// Minimum TypeScript Version: 3.8
```
Comment (Methuselah96) [inline types/three/index.d.ts:6]: Why is this change necessary?

Comment (joshuaellis) [reply types/three/index.d.ts:6]: Because doing `export * as MathUtils from 'PATH'` fails the test suite @ 3.6 & 3.7 for some reason.

Comment (Methuselah96) [reply types/three/index.d.ts:6]: 👍 Looks like it's due to `export * as` syntax being added in 3.8 (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#export--as-ns-syntax).

Comment (Methuselah96) [reply types/three/index.d.ts:6]: Looks like you would have to bump the minimum required version of `aframe` as well though. Maybe it's better to just stick with this for now?:
```
import * as MathUtils from './math/MathUtils';
export { MathUtils };
```

Comment (joshuaellis) [reply types/three/index.d.ts:6]: Yeah, the issue is aframe min version is 3.6, I'm not sure the process to resolve this?

Comment (joshuaellis) [reply types/three/index.d.ts:6]: Yeah just seen that recommended on Discord, will do that tonight

**Review by Methuselah96 — APPROVED** [review]
